Xamarin.Forms.Platform.UAP/CollectionView/ScrollHelpers.cs

```diff
@@ -0,0 +1,338 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using UWPPoint = Windows.Foundation.Point;
+using UWPSize = Windows.Foundation.Size;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+	internal static class ScrollHelpers
+	{
+		static UWPPoint Zero = new UWPPoint(0, 0);
+
+		static bool IsVertical(ScrollViewer scrollViewer)
+		{
+			return scrollViewer.HorizontalScrollMode == ScrollMode.Disabled;
```

Comment:
Note entirely up-to-date with UWP but is this enough to determine with certainty that it's vertical? There is also a VerticalScrollMode, don't we need to use that or at least get that into the equation?